zerver/webhooks/stripe/view.py

```diff
@@ -123,7 +129,8 @@ def default_body(update_blacklist: List[str]=[]) -> str:
             body = 'Discount {verbed} ([{coupon_name}]({coupon_url})).'.format(
                 verbed=event.replace('_', ' '),
                 coupon_name=object_['coupon']['name'],
-                coupon_url='https://dashboard.stripe.com/{}/{}'.format('coupons', object_['coupon']['id'])
+                coupon_url='https://dashboard.stripe.com/{}/{}'.format(
+                    'coupons', object_['coupon']['id'])
```

Comment:
Even for changes like this, again, it is better to include them in a separate commit. When you make a PR, it is usually great to have every commit focus on one aspect of the PR (i.e. adding support for `payout` events) and not make additional changes here and there unless they pertain to the feature you are implementing. It makes the PR easier to review because I can be sure that whatever was changed was changed because it was essential to the project at hand.

Also, when it comes to lines of code, in Zulip they are wrapped at 110 characters (I think), and not at 80, so clean-up like this is usually never done/needed!